study-builder/tenants/circadia/emails/welcome_email.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1"
+    />
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+            href="https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;700&display=swap"
+            rel="stylesheet"
+    />
+
+    <!--[if !mso]><!-->
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <title>Circadia - Account Created</title>
+    <!--<![endif]-->
+
+    <!--[if (gte mso 9)|(IE)]>
+    <xml>
+        <o:OfficeDocumentSettings>
+            <o:AllowPNG />
+            <o:PixelsPerInch>96</o:PixelsPerInch>
+        </o:OfficeDocumentSettings>
+    </xml>
+    <![endif]-->
+
+    <!--[if (gte mso 9)|(IE)]>
+    <style type="text/css">
+        body {
+          width: 600px;
+          margin: 0 auto;
+        }
+        table {
+          border-collapse: collapse;
+        }
+        table,
+        td {
+          mso-table-lspace: 0pt;
+          mso-table-rspace: 0pt;
+        }
+        img {
+          -ms-interpolation-mode: bicubic;
+        }
+      </style>
+    <![endif]-->
+
+    <style type="text/css">
+      body,
+      p,
+      div {
+        font-family: Nunito, sans-serif;
+        font-size: 16px;
+        color: #282828;
+      }
+
+      p {
+        margin: 0 0 20px 0;
+        padding: 0;
+      }
+    </style>
+</head>
+<body>
+<!-- Variable definitions -->
+{% assign logo_url = "https://storage.googleapis.com/circadia-dev-assets/auth0/logo.png" %}
```

Comment:
I recommend pulling out `circadia-dev-assets` into a substitution. Add it to Vault and render it in `circadia-config.json.ctmpl`. Or maybe we can use `##ASSETS_URL##` like how we have it in `password_reset.html`?

Reply:
@yufengwng thank you, I'll handle it.